cmd/brew-upgrade-mysql

```diff
@@ -43,11 +45,22 @@ EOM
 if [[ -d $(brew --prefix mysql@5.6) || -d $(brew --prefix mysql@5.7) ]]; then
   if [[ -d $(brew --prefix mysql@5.6) ]]; then
     brew services stop mysql@5.6 || true
+
+    while ! [[ "$(pidof mysqld)" == "" ]]; do
```

Comment:
@tarebyte FYI: `pidof` isn't a command provided by macOS. On my machine I get: `vendor/homebrew-bootstrap/cmd/brew-upgrade-mysql: line 49: pidof: command not found`. Instead, how about `pgrep mysqld`?